themes/root/templates/Helpers/icons/font.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+    $this->headLink()->appendStylesheet($this->src);
+
+    // Append extra classes
+    if (isset($this->extra['class'])) {
+        $this->icon .= ' ' . $this->extra['class'];
+        $this->attrs = preg_replace('/ class="[^"]+"/', '', $this->attrs);
+    }
+?>
+<span class="icon--font <?=$this->icon ?>"<?=$this->attrs ?> role="img" aria-hidden="true"></span>
```

Comment:
Should we `escapeHtmlAttr` the `$this->icon` value here, just to be on the safe side?